homeassistant/components/controller/__init__.py

```diff
@@ -0,0 +1,63 @@
+"""
```

Comment:
I'm not sure if there is a specific need for making controllers a platform in a specific component. Not that I think it is, by any means, a bad idea. I'm just not sure we should mix things up without a solid reason. Definitely a good topic to discuss though.